src/init.cpp

```diff
@@ -1249,10 +1249,10 @@ bool AppInitMain(InitInterfaces& interfaces)
             // and because this needs to happen before any other debug.log printing
             LogInstance().ShrinkDebugFile();
         }
-        if (!LogInstance().OpenDebugLog()) {
+    }
+    if (!LogInstance().StartLogging()) {
             return InitError(strprintf("Could not open debug log file %s",
```

Comment:
https://github.com/bitcoin/bitcoin/pull/16112/commits/2b0f4e1726379150fd3fc1e3f7bc127303ad850b

nit, could fix indentation.

Reply:
Done

Reply:
Leaving it to minimize the diff